src/config.py

```
@@ -0,0 +1,60 @@
+    cache_enabled: bool = True
+    cache_ttl_seconds: int = 3600  # 1 hour
+    
+    # Monitoring
+    monitor_check_interval_minutes: int = 30
+    monitor_max_concurrent_checks: int = 5
+    
+    # Logging
+    log_level: str = "INFO"
+    log_file: Optional[str] = "ch11.log"
+    
+    def __init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)